substation.go

```diff
@@ -10,6 +11,9 @@ import (
 	"github.com/brexhq/substation/v2/transform"
 )
 
+//go:embed substation.libsonnet
+var Libsonnet string
```

Comment:
These are referred to by Jsonnet as "libraries," so renaming the var to `Library` would make a bit more sense semantically. (Also gives us slightly better forward compatibility in case file extensions change for some reason.)